site/content/en/docs/admission-check-controllers/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Admission Checks"
-linkTitle: "Admisssion Checks"
+linkTitle: "Admission Checks"
```

Comment:
non-blocking ask: if we have one typo like this we may have more. Would you like to run a spell checker across the docs and confirm this is all? In case we have more feel free to open a follow up, or extend this PR.

Reply:
The `Admisssion` typo seems to exist only here :https://github.com/search?q=repo%3Akubernetes-sigs%2Fkueue%20Admisssion&type=code

Reply:
I'm ok with merging this PR.
As you say, I think that we can look for any typos in other PRs.

Reply:
Thanks for the quick review and merge. Here is the follow-on PR which addresses typos on ALL `.md` files.

https://github.com/kubernetes-sigs/kueue/pull/1748